torch/csrc/autograd/functions/convolution.cpp

```diff
@@ -727,6 +727,11 @@ auto ConvBackwardBackward::apply(const variable_list& grad_grad_inputs) -> varia
       gI = apply_fn<Transpose>(0, 1)(gIt);
     }
   }
+
+  if (!ggO.defined()) ggO = at::zeros_like(gO);
```

Comment:
you could check !defined && should_compute_output of these, right?

Reply:
Yes, that's true. Will patch in a sec